app/models/importers/eligible_fe_providers.rb

```diff
@@ -0,0 +1,55 @@
+require "csv"
+
+class Importers::EligibleFeProviders
```

Comment:
Take a look at the `CsvImporter::Base` class which the other importers inherit from. This was introduced to consolidate several differing implementations so it would be preferable to use this.

Reply:
The new class now inherits from `CsvImporter::Base`. I've had to make some minor changes. Out of scope of this PR but we should probably namespace this grouping of work. Another nice to have would be to make it `Actvemodel::Errors` compliant which current it is not